.github/workflows/deploy-public-build.yml

```diff
@@ -1,11 +1,12 @@
 # Adapted from https://github.com/marketplace/actions/deploy-pr-preview and
 # https://github.com/sitek94/vite-deploy-demo
-name: Deploy Public Build
+name: Deploy GH Pages Build
 
 on:
   push:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
Allow manual triggering (though this can also be run via `npm run deploy`)

Reply:
This ended up being helpful for troubleshooting the GH action when it broke after this PR was merged 😅